challangeNetflix/DetailsViewController.swift

```diff
@@ -132,13 +132,13 @@ class DetailsViewController: UIViewController {
 
     }
     
-    func favExist (title: String) -> Bool {
+    func favExist (id: String) -> Bool {
         
         do {
             
             let realm = try Realm()
             
-            let objects = realm.objects(MovieRealm.self).filter("title = \"\(title)\"")
+            let objects = realm.objects(MovieFavRealm.self).filter("id = \"\(id)\"")
             
             return objects.count > 0
```

Comment:
```suggestion
            return realm.object(ofType: MovieFavRealm.self, forPrimaryKey: id) != nil
```

Dessa forma tu não procura por todos objetos do banco, e sim tu pega diretamente o objeto que tu quer pelo primaryKey, é mais performatico fazer dessa forma e fica mais simples